### PR TITLE
updated README, package.json and app.json.

### PR DESCRIPTION
**Pull request by kachmashk**

### CODE REVIEW

**Comment by kachmashk** [comment]
* **#9** <a href="https://app.graphite.dev/github/pr/2gather-app/togather2/9?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kachmashk and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: app/Event/index.tsx [hunk 1]
```diff
@@ -1,66 +1,79 @@
...
 
-  const onBackPress = () => console.log("back pressed");
-  const onJoinPress = () => console.log("join press");
+  const onBackPress = () => console.log('back pressed')
+  const onJoinPress = () => console.log('join press')
   const onCheckoutParticipantsPress = () =>
-    console.log("checkout participants press");
+    console.log('checkout participants press')
+
+  const [state, setState] = useState(0)
```
Comment (bionanek) [inline app/Event/index.tsx:51]: this and useeffect below seems to be added by accident?

File: app/Event/index.tsx [hunk 2]
```diff
@@ -1,66 +1,79 @@
...
       <Stack.Screen options={{ headerShown: false }} />
       <TopBar
-        eventCategory="Sport"
+        eventCategory='Sport'
         isFavourite={isFavourite}
         onFavouritePress={handleOnFavouritePress}
         onBackButtonPress={onBackPress}
       />
+
+      <Button
```
Comment (bionanek) [inline app/Event/index.tsx:72]: also shouldn't be pushed

**Review by bionanek — CHANGES_REQUESTED** [review]
remove dummy code and we're good to go

**Review by bionanek — APPROVED** [review]

**Comment by kachmashk** [comment]
### Merge activity

* **Jun 5, 3:33 AM EDT**: @kachmashk merged this pull request with [Graphite](https://app.graphite.dev/github/pr/2gather-app/togather2/9).

**Pull request merged**